### ISSUE TITLE
Lithium error trying to install on Windows IIS

### ISSUE THREAD

**Issue by mattenz** [nZEDb/nZEDb#2565]
I'm currently trying to get everything running on Server 2016 running IIS with PHP 7.1.  I've run through a combination of https://github.com/nZEDb/nZEDb/wiki/Install-Guide:-Windows-using-WAMP and https://github.com/nZEDb/nZEDb/wiki/Windows-IIS-with-FastCGI successfully right up to the point where I actually want to run the installer in the browser.  When I do so I get the following error:

> [04-Jan-2018 07:30:17 UTC] PHP Fatal error:  Uncaught lithium\core\ConfigException: The data connection `default` is not configured for model `app\models\Settings`. in C:\inetpub\nZEDb\libraries\lithium\data\Model.php:1470
Stack trace:
#0 C:\inetpub\nZEDb\libraries\lithium\data\Model.php(659): lithium\data\Model::connection()
#1 C:\inetpub\nZEDb\app\models\Settings.php(176): lithium\data\Model::lithium\data\{closure}(Array)
#2 C:\inetpub\nZEDb\libraries\lithium\data\Model.php(679): app\models\Settings::app\models\{closure}(Array, Object(Closure))
#3 C:\inetpub\nZEDb\libraries\lithium\aop\Filters.php(231): lithium\data\Model::lithium\data\{closure}(Array)
#4 C:\inetpub\nZEDb\libraries\lithium\data\Model.php(682): lithium\aop\Filters::run('app\\models\\Sett...', 'find', Array, Object(Closure))
#5 C:\inetpub\nZEDb\app\models\Settings.php(252): lithium\data\Model::find('setting', Array)
#6 C:\inetpub\nZEDb\app\config\bootstrap\yenc.php(44): app\models\Settings::value('apps..yydecoder...', true)
#7 C:\inetpub\nZEDb\app\config\bootstrap.php(92): require_once( in C:\inetpub\nZEDb\libraries\lithium\data\Model.php on line 1470

**Comment by niel** [comment]
Strange, you shouldn't see that error during the first 3 steps of installing. Can you give more details of which step this is at?

**Comment by niel** [comment]
Looks like those guides are woefully out of date. Have you installed Composer  (https://github.com/nZEDb/nZEDb/wiki/Guides:-composer) and run the composer install process?

**Comment by mattenz** [comment]
Yeah, I did do that part - the only issue I have with Composer is at the end it gives me this message:
> nzedb\build\Composer::postInstallCmd
Updating git hooks... 'build' is not recognized as an internal or external command,
operable program or batch file.

Initially I went through only the IIS guide and when I first went to launch it, it was complaining about pages in the lithium directory being missing so after going through the others I managed to piece together what was missing and go forward.  After comparing between the various Linux and Windows guides I can't see anything that I haven't completed.

I'm not really getting to any steps.  Whether I go to http://host/Install or just http://host I only ever see that error message.

**Comment by niel** [comment]
Would you try copying .../configuration/settings.example.php to .../configuration/settings.php please. That will set up some basic options, including error logging. Once that is done try accessing the \install address again and check the error logs in .../resources/logs/ for additional information.

**Comment by mattenz** [comment]
Thanks!  That did the trick.

I then found there were a few other things I needed to do, such as loading the timezone info from a SQL file.  However, I just got through all the install pages and the Admin and main page are now loading.  Now to get the actual indexing processing and maybe I'll then look at converting over my existing 5 years of Newznab data across.

If you think it's worthwhile, I could go through and write an updated guide for IIS?
